app/services/usps_in_person_proofing/transliterable_validator.rb

```diff
@@ -87,5 +94,15 @@ def get_invalid_chars(value)
       # Create sorted list of unique unsupported characters
       (result.unsupported_chars + additional_chars).sort.uniq
     end
+
+    def analytics(user: AnonymousUser.new)
+      Analytics.new(user: user, request: nil, session: {}, sp: nil)
+    end
+
+    def log_nontransliterable_characters(result)
+      analytics.idv_in_person_proofing_nontransliterable_characters_submitted(
+        nontransliterable_characters: result,
+      )
+    end
```

Comment:
maybe we just inline this method to its callsite? since it's just wrapping/renaming the argument variable?

Reply:
good point